src/Yasumi/Provider/Canada.php

```diff
@@ -137,7 +138,8 @@ protected function calculateNationalIndigenousPeoplesDay(): void
     /**
      * Canada Day.
      *
-     * @see https://en.wikipedia.org/wiki/Canada_Day
+     * @see https://en.wikipedia.org/wiki/Canada_Day and Holidays Act, R.S.C., 1985, c. H-5
```

Comment:
If possible, can you add a link to that Holidays Act?

Reply:
added in a further commit